index.d.ts

```diff
@@ -2506,15 +2521,15 @@ declare namespace Eris {
 
   export class ComponentInteraction<T extends PossiblyUncachedTextable = TextableChannel> extends Interaction {
     channel: T;
-    data: ComponentInteractionButtonData | ComponentInteractionSelectMenuData;
+    data: ComponentInteractionButtonData | ComponentInteractionSelectMenuData | unknown;
```

Comment:
Why was unknown added here?

Reply:
This makes using the property impossible without explicitly casting to one of the two

![](https://butts-are.cool/Code_-_Insiders_09-26-2021_14-36-55.png)